packages/reactor-browser/package.json

```diff
@@ -9,45 +9,46 @@
   "files": [
     "dist"
   ],
-  "types": "dist/src/index.d.ts",
-  "main": "dist/index.js",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
+    }
+  },
   "scripts": {
     "check-types": "tsc",
     "lint": "eslint .",
     "lint:nx": "eslint . --fix --quiet",
     "lint:fix": "eslint --fix",
     "postlint": "npm run check-types",
-    "build": "NODE_ENV=production vite build",
-    "build:watch": "vite build --watch",
+    "build": "tsup",
+    "build:watch": "tsup --watch",
     "test": "vitest run --passWithNoTests",
     "clean": "rimraf dist",
     "clean:node_modules": "rimraf node_modules"
   },
   "keywords": [],
   "author": "",
   "devDependencies": {
-    "@powerhousedao/config": "workspace:*",
-    "@powerhousedao/design-system": "workspace:*",
-    "@powerhousedao/scalars": "workspace:*",
-    "document-drive": "workspace:*",
-    "document-model": "workspace:*",
-    "document-model-libs": "workspace:*",
     "@types/react": "^18.3.1",
     "@types/react-dom": "^18.3.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1"
   },
   "peerDependencies": {
-    "@powerhousedao/config": "workspace:*",
-    "@powerhousedao/design-system": "workspace:*",
-    "@powerhousedao/scalars": "workspace:*",
-    "document-drive": "workspace:*",
-    "document-model": "workspace:*",
-    "document-model-libs": "workspace:*",
     "react": "^18.3.1",
     "react-dom": "^18.3.1"
   },
   "dependencies": {
+    "@powerhousedao/config": "^1.5.0",
```

Comment:
I think these should still use the workspace:* protocol. putting them here in dependencies should still work the same, but the versions will be kept in sync

Reply:
added fix here: [eb85db7](https://github.com/powerhouse-inc/powerhouse/pull/802/commits/eb85db7fda5a7e7f15fe6946cc81a6742fecef41)